template/envs/dev.env

```diff
@@ -0,0 +1,2 @@
+NODE_ENV=development
+LOGLEVEL=debug
```

Comment:
Should this be ```LOG_LEVEL```? ```test.env``` has ```LOG_LEVEL```

Reply:
Should be `LOG_LEVEL` because `LOGLEVEL` env var is used by moleculer.